src/main/scala/edu/twitter/model/TweetsLoader.scala

```diff
@@ -0,0 +1,45 @@
+package edu.twitter.model
+
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.{DataFrame, Dataset, Row}
+
+/**
+  * Loader loads all tweets model data this data is static data in FS not HDFS.
+  */
+class TweetsLoader {
+  private val tweetsJsonParser = new JsonParser("local[*]", "Twitter");
+
+  /**
+    * Get Tweets data set as RDD.
+    * @return RDD contains stream of tweets
+    */
+  def getTweetsDataSet(): RDD[Row] = {
+
+    var tweetDF = tweetsJsonParser.parse("tweets\\*")
```

Comment:
can we move tweets under project's resources, "src/main/resources"

Reply:
yep we can